spec/factories/virtual_machine_templates.rb

```diff
@@ -1,10 +1,11 @@
 FactoryGirl.define do
   factory :virtual_machine_template, aliases: [:source_template], class: 'Atmosphere::VirtualMachineTemplate' do |f|
-    tenant
     name { FFaker::Lorem.characters(5) }
     id_at_site { FFaker::Internet.ip_v4_address }
     state :active
 
+    tenants { [FactoryGirl.create(:tenant)] }
```

Comment:
`build` instead of `create` should be enough there. The reason for it is that some tests are not actually create object in DB only as AR objects.